cmd/podman/kube/play.go

```diff
@@ -139,6 +139,9 @@ func playFlags(cmd *cobra.Command) {
 
 	downFlagName := "down"
 	flags.BoolVar(&playOptions.Down, downFlagName, false, "Stop pods defined in the YAML file")
+	if cmd == playCmd {
+		_ = flags.MarkHidden("down")
+	}
```

Comment:
Do not add the flag in the first place for the down command. Also you have to check for playKubeCmd not playCmd.

Reply:
The flag is still present for both "kube play" and "kube down". I'm also unsure as to why I would check for playKubeCmd as this would hide the flag for "play kube" when the aim is to only hide it for "kube play".

Reply:
I think it should be hidden for both `play kube` and `kube play`.  This will keep things simple and hopefully direct users to `kube down`.